mentorship/slack_coc.md

```diff
@@ -0,0 +1,70 @@
+# R-devel Slack
+
+The R-devel Slack is a supportive space to foster the community of R developers contributing to R core:
+
+- **Potential/novice contributors** wanting support to get started/develop as contributors.
+- **Experienced contributors and members of R Core** willing to provide ad-hoc support.
+- **Community organizers** wanting to connect their communities with this initiative.
+
+By "R core" we refer to the [base packages](https://cran.r-project.org/doc/FAQ/R-FAQ.html#Which-add_002don-packages-exist-for-R_003f), which R users outside of the [R Core Team](https://www.r-project.org/contributors.html) may contribute to through discussion of bugs/new features, [reviewing bug reports](https://developer.r-project.org/Blog/public/2019/10/09/r-can-use-your-help-reviewing-bug-reports/index.html) and proposing patches. On the Slack we will discuss coding issues, the wider development process and outreach initiatives. 
+
+## Posting Guide
+
+To create a welcoming and supportive workspace, we ask members to:
+- Be welcoming, friendly and patient.
+- Avoid sarcasm or belittling language.
+- Respect people's technology choices.
+- Provide direct links vs "RTFM"/"Just Google it" responses.
+- Use gender-neutral language.
```

Comment:
I wonder if it would be helpful to include a resource link here about gender-neutral language? I don't know if there's a "gold-standard" resource but I can look into it, and/or happily defer to someone else's expertise.

Reply:
A good idea - it is actually not that easy to find something that isn't overly-detailed or aimed at a specific audience with a lot of content that is not very relevant to this Slack. But I found a fact sheet that covers the main points reasonably well and added this to the bullet point "(see e.g. this [factsheet on gender inclusive language](https://services.anu.edu.au/human-resources/respect-inclusion/gender-inclusive-language))." For me the main points are
- use "they" when talking about a hypothetical person
- avoid using "guys"
- avoid using terms such as "manpower"

Reply:
Thanks for looking into this Heather! I like the fact sheet that you found and the idea of including some bullet points 👍